samples/snippets/csharp/VS_Snippets_CLR_System/system.Collections.Generic.HashSet_ExceptWith/cs/source2.cs

```diff
@@ -86,7 +86,7 @@ public override bool Equals(string s1, string s2)
 
     public override int GetHashCode(string s)
     {
-        return base.GetHashCode();
+        return s.ToLower().GetHashCode();
```

Comment:
This will crash if `s == null`; a better implementation is to use `StringComparer.OrdinalIgnoreCase`: #4295